docs/content/en/overview/authentication/security-key/index.md

```diff
@@ -1,51 +1,52 @@
 ---
-layout: default
-title: Security Keys
-nav_order: 2
-parent: Second Factor
-grand_parent: Features
+title: "Security Key"
+description: "Authelia utilizes WebAuthn security keys as one of it's second factor authentication methods."
+lead: "Authelia utilizes WebAuthn security keys as one of it's second factor authentication methods."
+date: 2022-03-20T12:52:27+11:00
+lastmod: 2022-06-03T10:43:55+10:00
+draft: false
+images: []
+menu:
+  overview:
+    parent: "authentication"
+weight: 240
+toc: true
+aliases:
+  - /docs/features/2fa/security-key
 ---
 
-# Security Keys
-
-**Authelia** supports hardware-based second factors leveraging [FIDO2]&nbsp;[Webauthn] compatible security keys like
+__Authelia__ supports hardware-based second factors leveraging [FIDO2]&nbsp;[WebAuthn] compatible security keys like
```

Comment:
```suggestion
__Authelia__ supports hardware-based second factors leveraging [FIDO2] [WebAuthn] compatible security keys like
```

Reply:
This is needed due to an issue with linking two things next to each other, let me see if I can resolve it though